### DISCUSSION TITLE
Is it possible to use this library on an Arduino?

### DISCUSSION THREAD

**Discussion by Raj2032** [gpiozero/gpiozero#1056] [General]
If I made something for the raspberry Pi and I wanted to also port my code over tot he Arduino, I don't want to rewrite everything. Is it possible to port my gpiozero code to the Arduino as well?

**Comment by lurch** [comment]
Nope, they're entirely different frameworks. Not least of which is that GpioZero uses Python (an interpreted language), but Arduino uses C (a compiled language, with different syntax) !

**Reply by Raj2032** [reply]
I see mate thanks mate, do you know anything that will allow me to write once run everywhere?

**Reply by lurch** [reply]
I dunno which "Arduino" you're using, but you might want to look at https://circuitpython.org/ ? :shrug:
